packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -351,32 +383,34 @@ class Cursor extends RootedTextCursor implements ITreeSubscriptionCursor {
             0x33b /* Cursor must not be freed */,
         );
         this.state = ITreeSubscriptionCursorState.Cleared;
-        this.keyStack.length = 0;
-        this.siblingStack.length = 0;
-        this.indexStack.length = 0;
-        this.index = -1;
+        this.innerCursor = undefined;
         this.forest.currentCursors.delete(this);
     }
 
-    public set(root: DetachedField, index: number): void {
+    public set(): void {
```

Comment:
I think this also needs a comment. Hard to tell set means when reading through the ObjectForest code.

Reply:
done. Also renamed.